src/Sentry.EntityFramework/Sentry.EntityFramework.csproj

```diff
@@ -21,8 +25,8 @@
     <Using Include="Sentry.Integrations" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net462'">
-    <PackageReference Include="EntityFramework" Version="6.2.0" />
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
```

Comment:
the highest supported version of EntityFramework is net45 - why do we build for net6.0 at all?

Reply:
> the highest supported version of EntityFramework (and log4net) is net45 - why do we target net6.0 at all?

Because I didn't know that 😉 I spent too long as a manager, I think... I missed the whole bit between .net core and .net 7.

Reply:
Just checking, so we don't want to turn on `IsAotCompatible` for `Sentry.EntityFramework `? That's the implication of not targeting `net6.0`. People may still be able to include it in applications that they publish AOT, but we wouldn't ever get any compiler assurance that that was the case. By turning on `IsAotCompatible` (which would require targeting `net6.0`) we get compiler warnings/errors if anything changes in this library such that it would break AOT compatibility.

Reply:
Hm, unlikely the MS package would work with AOT. I'll revert this one.

Reply:
Yeah EntityFramework (aka EF6) is "old". EF Core is the "new" thing. We don't need to invest in AOT for EF 6